fetch/main.js

```diff
@@ -0,0 +1,3 @@
+fetch('https://jsonplaceholder.typicode.com/users')
+.then(Response => Response.json())
```

Comment:
It is best to have parameters start with a lowercase letter, so here `response` would be better